src/lm/mikolov-rnnlm-lib.h

```diff
@@ -0,0 +1,320 @@
+///////////////////////////////////////////////////////////////////////
+// 
+//  Recurrent neural network based statistical language modeling toolkit
+//  Version 0.3e
+//  (c) 2010-2012 Tomas Mikolov (tmikolov@gmail.com)
+//           2015 Guoguo Chen
+//           2015 Hainan Xu
+// 
+///////////////////////////////////////////////////////////////////////
+
```

Comment:
Guoguo, we need to have some more clarity on the license here.
What did the LICENSE file in his distribution say?